assessment.sql

```diff
@@ -5,24 +5,49 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees(
+  id SERIAL PRIMARY KEY
+  surname CHARACTER VARYING
+  given_name CHARACTER VARYING
+  gender CHARACTER VARYING
+  height INTEGER
+  weight INTEGER
+  dob DATE
+  eye_color CHARACTER VARYING
+  hair_color CHARACTER VARYING
+);
```

Comment:
Commas must be used to separate the column names and types from one another. Otherwise, this is a good statement. 3/4